src/ui/tabview/TabController.qml

```diff
@@ -50,11 +50,24 @@ QtObject {
     signal fullScreenRequested(var request)
     signal openUrlInNewPrivateWindowRequested(var url)
 
-    function openUrl(url, background) {
-        addTab(TabType.fromUrl(url), {
-            url: url,
-            background: background
-        });
+    function completeTabsRestore()
+    {
+        for (var i = 0; i < tabContentView.pages.count; ++i)
+        {
+            var tabPage = tabContentView.pages.get(i).item;
```

Comment:
Let's stay consistent with the rest of the JavaScript code and keep the opening curly brace in the same line ;)
```js
for (var i = 0; i < tabContentView.pages.count; ++i) {
```